src/panels/calendar/confirm-event-dialog-box.ts

```diff
@@ -0,0 +1,145 @@
+import "@material/mwc-button/mwc-button";
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { fireEvent } from "../../common/dom/fire_event";
+import "../../components/ha-dialog";
+import "../../components/ha-svg-icon";
+import "../../components/ha-switch";
+import { HomeAssistant } from "../../types";
+import { ConfirmEventDialogBoxParams } from "./show-confirm-event-dialog-box";
+import { RecurrenceRange } from "../../data/calendar";
+
+@customElement("confirm-event-dialog-box")
+class ConfirmEventDialogBox extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() private _params?: ConfirmEventDialogBoxParams;
+
+  public async showDialog(params: ConfirmEventDialogBoxParams): Promise<void> {
+    this._params = params;
+  }
+
+  public closeDialog(): boolean {
+    return true;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._params) {
+      return html``;
+    }
+
+    return html`
+      <ha-dialog
+        open
+        ?scrimClickAction="true"
+        ?escapeKeyAction="true"
+        @closed=${this._dialogClosed}
+        defaultAction="ignore"
+        hideActions="false"
+        .heading=${this._params.title}
+      >
+        <div>
+          <p>${this._params.text}</p>
+        </div>
+        <mwc-button @click=${this._dismiss}>
+          ${this.hass.localize("ui.dialogs.generic.cancel")}
+        </mwc-button>
+        <mwc-button
```

Comment:
```suggestion
        <mwc-button slot="primaryAction"
```

Reply:
After fixing this, it looks like this:

<img width="618" alt="Screen Shot 2022-10-17 at 8 51 58 PM" src="https://user-images.githubusercontent.com/6026418/196331526-497985e6-d169-455f-9178-9861d14be741.png">

Is that red outline OK?

(I also removed the `hideActions` above)

Reply:
Yeah, the red outline is ok, would maybe only swap `all` and `only this` buttons